src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,113 @@
+import java.util.*;
+import java.lang.String;
+
+
 public class StringManipulation implements StringManipulationInterface {
 
+    private String manipulatedstring;
     @Override
     public String getString() {
-        return null;
+        return manipulatedstring;
     }
 
     @Override
-    public void setString(String string) {
-    }
+    public void setString(String string) { manipulatedstring = string;}
+
 
     @Override
     public int count() {
-        return 0;
-    }
+        int wordCount = 0;
+        if (manipulatedstring == "") {
+            return wordCount;
+        }
+        String trimmedString = manipulatedstring.trim();
```

Comment:
Seems like trim is a popular tool for this method implementation, looks good to me at the moment.